### PR TITLE
Changed reducer syntax. Updated README

### PR DESCRIPTION
**Pull request by f1yn**
I changed the syntax of the `create_store` method to `createStore`, as well as updated the README. I did this to better match the current spec of Redux. Update: Also updated `with_logger` to `withLogger`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -235,10 +235,10 @@ changes to the console.  To use it, simply decorate your reducer with the
...
 
 ```javascript
-import { create_store } from "innerself";
+import { createStore } from "innerself";
 import with_logger from "innerself/logger";
 import reducer from "./reducer"
 
 const { attach, connect, dispatch } =
-    create_store(with_logger(reducer));
+    createStore(with_logger(reducer));
```
Comment (stasm) [inline README.md]: For consistency, let's rename this to `withLogger`, please.

File: README.md [hunk 2]
```diff
@@ -235,10 +235,10 @@ changes to the console.  To use it, simply decorate your reducer with the
 default export of the `innerself/logger` module:
 
 ```javascript
-import { create_store } from "innerself";
+import { createStore } from "innerself";
 import with_logger from "innerself/logger";
```
Comment (stasm) [inline README.md]: For consistency, let's rename this to `withLogger`, please.

**Review by stasm — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (f1yn) [reply README.md]: Alright. I'll push another commit

Comment (f1yn) [reply README.md]: Changed `with_logger` to `withLogger`

**Pull request merged**